common/lib/xmodule/xmodule/js/src/sequence/display.coffee

```diff
@@ -161,10 +162,15 @@ class @Sequence
 
       # update the data-attributes with latest contents only for updated problems.
       if @anyUpdatedProblems new_position
-        $.each @updatedProblems[new_position], (problem_id, latest_content) =>
+        $.each @updatedProblems[new_position], (problem_id, latest_data) =>
+          latest_content = latest_data[0]
```

Comment:
Nit: is this necessary?  You only use latest_content once below to update the content data.